docs/test-fairy/testing-an-app/bug-tracking/jira-cloud.md

```diff
@@ -1,17 +1,129 @@
+   In case you get a **PENDING** response, check the connection configuration.
+   <img src={useBaseUrl('/img/test-fairy/testing-an-app/bug-tracking/jira-connect-test-ok.png')} alt="Connection configuration"/>
+   :::
+
+   Each **Issue type** has different fields associated with it:
+
+   - **Name** - the field name.
+   - **Type** - the type of field as defined in the JIRA system.
+   - **Value** - the values from the JIRA system (in addition to predefined fixed values and Dynamic value, see below)
+   - **Required?** - specify if the field required or optional.
+     <img src={useBaseUrl('/img/test-fairy/testing-an-app/bug-tracking/jira-requiered-fildes-mark.png')} alt="Required fields"/>
+
+When defining fields in the Configure Fields window, follow the below conventions:
+
+- When you choose a field from a drop-down list, this field as is (text) will be populated in the corresponding JIRA field in the issue opened.
+  <img src={useBaseUrl('/img/test-fairy/testing-an-app/bug-tracking/jira-values-drop-down.png')} alt="Values drop down"/>
+
+- You can also choose from the fixed predefined values in the list below:
+  - `{user.id}` - the UserId of the session running.
+  - `{session.timestamp}` - the timestamp of the session.
+  - `{session.url}` - the session URL on the TestFairy dashboard.
+  - `{session.ipAddress}` - the IP address of the device running the session.
+  - `{device.os}` - the running device OS
+  - `{device.model}` - the device model of the handset
+  - `{device.osVersion}` - the OS version on the device (if the iPhone is running version IOS 12 value=12)
+  - `{app.name}` - the app name.
+  - `{app.version}` - the _versionName_ or _CFBundleShortVersionString_ of the build. example: 1.7.0
+  - `{app.fullVersion}` - the _versionName_ + (_versionCode_ or _CFBundleVersion_) of the build. example: 1.7.0 (1700)
+  - `{feedback.text}` - the feedback message
+  - `{feedback.timestamp}` - the timestamp of the feedback (absolute)
+  - `{feedback.relTimestamp}` - relative timestamp of the feedback (mm:ss) since the session started
+
+To use these values, add them to the `Dynamic value` field that opens when you select the **Dynamic value** option like so:
+<img src={useBaseUrl('/img/test-fairy/testing-an-app/bug-tracking/jira-fixed-attr-popup.png')} alt="Fixed values"/>
+
+- You can add attributes defined in your app's code to the `Dynamic value` field. The structure of a dynamic field is as follows:
+  `{attr.[attribute_name]||[default_value]}`.
+- `attribute_name` - is the name of the Teasfairy attribute set in the code by the `TestFairy.setAttribute` function. What passes to the JIRA is the value of this attribute.
+- `default_value` - for each attribute, you can set a default value so that if you receive a null or wrong attribute value from the code (not possible in this field in JIRA), the default value will be passed to JIRA instead.
```

Comment:
```suggestion
- `default_value` - for each attribute, you can set a default value so that if you receive a null or wrong attribute value from the code (impossible in this field in Jira), the default value will be passed to Jira instead.
```